pep-0013.rst

```diff
@@ -21,13 +21,13 @@ Current steering council
 
 The current steering council consists of:
```

Comment:
Would it be better to put in the year?
```suggestion
The 2022 steering council consists of:
```

Reply:
I'd probably leave it out, we link to the relevant PEP and it's one more thing to forget to update :)

But it's right next to the other things needing change so no strong opinion.

Reply:
Agree with Hugo, for what it's worth.
A

Reply:
My first intuition was also that its more churn, but I think I agree with @encukou that its better to have it so if it _is_ out of date, it is immediately clear to readers and ourselves what year it is for, and they don't confused and not know which year the members listed are for without having to read the whole voting PEPs. So better to be explicit than implicit here, IMO.